extensions/wikia/AdEngine3/src/bill-the-lizard-wrapper.js

```diff
@@ -6,13 +6,16 @@ import {
     context,
     events,
     eventService,
+    slotService,
     utils
 } from '@wikia/ad-engine';
 import targeting from './targeting';
 
+const AD_SLOT_CATLAPSED_STATUS = 'catlapsed';
```

Comment:
Can we just add this to `AdSlot` in AdEngine and re-use it here like we did for `AdSlot.SLOT_RENDERED_EVENT`?

Reply:
done